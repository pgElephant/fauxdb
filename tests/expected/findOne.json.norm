{
  "_id": "pg_1",
  "ok": 1
}
