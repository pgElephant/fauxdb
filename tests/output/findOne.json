{
  ok: 1,
  _id: '1',
  name: 'test_record_1',
  value: 10,
  active: 'f',
  created_at: 2025
}
