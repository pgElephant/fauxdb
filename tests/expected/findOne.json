{
  "ok": 1,
  "_id": "pg_1"
}
